test/integration/scenarios/index.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
 	network: {
 		peers: require('./network/peers'),
 		peerDisconnect: require('./network/peer_disconnect'),
+		peerBlacklist: require('./network/peer_blacklist'),
```

Comment:
To keep pragmatic in place. Please rename it to `peersBlackList`. as its a singular list containing plural peers.